### PR TITLE
BZ#1900889 and BZ# 1901149  Updates to VMM guide and Admin guide VM migration options

### PR DESCRIPTION
**Pull request by emarcusRH**
Fixes issue #1900889 and # 1901149

Changes proposed in this pull request:

- Update VMM guide and Admin guide to remove "custom migration downtime" and "auto converge migration" options, and add "enable migration encryption" option
- Update VMM guide and Admin guide to add/reorganize CPU options

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @emarcusRH )

This pull request needs review by: (please @RichardHoch )

### CODE REVIEW

File: source/documentation/administration_guide/topics/Virtual_Machine_Pool_Host_settings_explained.adoc [hunk 1]
```diff
@@ -14,6 +14,15 @@ The following table details the options available on the *Host* tab of the *New
...
 * *Specific Host(s)* - The virtual machine will start running on a particular host in the cluster. However, the {engine-name} or an administrator can migrate the virtual machine to a different host in the cluster depending on the migration and high-availability settings of the virtual machine. Select the specific host or group of hosts from the list of available hosts.
 
+|*CPU options*
+|*Pass-Through Host CPU*
+|When selected, allows virtual machines to use the host's CPU flags.
+When selected, *Migration Options* is set to *Allow manual migration only*.
+
+|
+|*Migrate only to hosts with the same TSC frequency*
+|When selected, this virtual machine can only be migrated to a host with the same TSC frequency. THis option is only valid for High Performance virtual machines.
```
Comment (dcdacosta) [inline source/documentation/administration_guide/topics/Virtual_Machine_Pool_Host_settings_explained.adoc:24]: Change to lower case h in "THis option".

File: source/documentation/administration_guide/topics/Virtual_Machine_Pool_Host_settings_explained.adoc [hunk 2]
```diff
@@ -22,32 +31,35 @@ The following table details the options available on the *Host* tab of the *New
...
++
+The post-copy policy first tries pre-copy to verify whether convergence can occur. The migration switches to post-copy if the virtual machine migration does not converge after a long time.
++
+This significantly reduces the downtime of the migrated virtual machine, and also guarantees that the migration finishes regardless of how rapidly the memory pages of the source virtual machine change. It is optimal for migrating virtual machines in heavy continuous use, which would not be possible to migrate with standard pre-copy migration.
++
+The disadvantage of this policy is that in the post-copy phase, the virtual machine may slow down significantly as the missing parts of memory are transferred between the hosts.
++
+[WARNING]
+====
+If the network connection breaks prior to the completion of the post-copy process, the {engine-name} pauses and then kills the running VM. Do not use post-copy migration if the VM availability is critical or if the migration network is unstable.
```
Comment (dcdacosta) [inline source/documentation/administration_guide/topics/Virtual_Machine_Pool_Host_settings_explained.adoc:50]: Suggest changing VM to virtual machine.

File: source/documentation/virtual_machine_management_guide/chap-Additional_Configuration.adoc [hunk 3]
```diff
@@ -117,4 +117,5 @@ include::topics/Configuring_high_performance_virtual_machines.adoc[leveloffset=+
 include::topics/Configuring_Time_Zones.adoc[leveloffset=+1]
 
 // Very basic attribute, since I assume this section could one day be its own assembly and this attribute will therefore not be needed.
-:vmm:
+//:vmm:  this attribute seems to be messing up the compilation - testing commenting it out - Eli Marcus
```
Comment (dcdacosta) [inline source/documentation/virtual_machine_management_guide/chap-Additional_Configuration.adoc:120]: This was not causing the issue; Said added a step 3 with an example to the Configuring_Time_Zones.adoc and forgot to include the closing ---- for the example which read everything following it as part of the example. I have corrected the issue in https://github.com/oVirt/ovirt-site/pull/2679 and assigned you as a reviewer.

**Review by dcdacosta — CHANGES_REQUESTED** [review]
A few minor comments; otherwise, LGTM.

**Review by RichardHoch — APPROVED** [review]

**Comment by RichardHoch** [comment]
LGTM!

**Review by dcdacosta — APPROVED** [review]

**Pull request merged**